webservice/src/main/java/dk/dbc/laesekompas/suggester/webservice/SearchResource.java

```diff
@@ -1,23 +1,6 @@
 package dk.dbc.laesekompas.suggester.webservice;
 /*
- * Copyright (C) 2019 DBC A/S (http://dbc.dk/)
```

Comment:
Strengt taget er det slet ikke nødvendigt med nogen form for header i kildetekstfilerne, så længe der ligger en passende COPYRIGHT.md i roden af projektet